### ISSUE TITLE
Type null is not a subtype of type int in subcast

### ISSUE THREAD

**Issue by douglasg14b** [liftoff-app/liftoff#205]
### Device info

- OS: Android
- OS version: 12
- Device: Pixel 6
- Liftoff version: 0.9.19

### Describe the bug

Fails to show this page, screenshot shows it pretty concisely 

![Screenshot_20230701-173549.png](https://github.com/liftoff-app/liftoff/assets/1400380/0948554a-cc53-48fe-a28e-84bc93d6458b)

### Steps to reproduce

Screenshot should say it all, idk what that tab is called...

### Relevant logs

> `no logs`

**Comment by zachatrocity** [comment]
Is this still happening? I believe this was due to instance outages

**Comment by swmarks** [comment]
> Is this still happening? I believe this was due to instance outages

This happened to me too when we changed API stuff for version v0.18 for sort stuff. I had to clear my app data to fix it.

**Comment by douglasg14b** [comment]
Yeah still happening for me. Either way still a bug since there would otherwise be error handling there.
